Mathlib/Algebra/Module/Equiv.lean

```diff
@@ -185,6 +185,10 @@ instance : SemilinearEquivClass (M ≃ₛₗ[σ] M₂) σ M M₂
   map_add := (·.map_add') --map_add' Porting note: TODO why did I need to change this?
   map_smulₛₗ := (·.map_smul') --map_smul' Porting note: TODO why did I need to change this?
 
+@[simp]
+theorem coe_mk {to_fun inv_fun map_add map_smul left_inv right_inv} :
```

Comment:
This needs an #align. Can you copy the exact lemma from mathport?

Reply:
This is also on the wrong line in the file:

https://github.com/leanprover-community/mathlib3port/blob/master/Mathbin/Algebra/Module/Equiv.lean#LL156-L160C46

Reply:
This is because coercions from `LinearEquiv`s to functions is generally implemented by `FunLike` and coercions are elaborated directly in Lean 4.